R/tm_expiring_contracts.R

```diff
@@ -35,12 +35,12 @@ tm_expiring_contracts <- function(country_name, contract_end_year, league_url =
     tryCatch({league_page <- .load_page(league_url)}, error = function(e) {league_page <- c()})
 
     tryCatch({country <- league_page %>%
-      rvest::html_nodes(".profilheader") %>%
+      rvest::html_nodes(".data-header") %>%
```

Comment:
Transfermarkt must have changed sopme of their html, primarily for the country and league names